components/Badges/badges.css

```diff
@@ -0,0 +1,98 @@
+@import "/reset.css";
+.box{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
Have a utility class for flex, so that you don't repeat it.

Reply:
ok done